nbodykit/cosmology/background.py

```diff
@@ -96,17 +96,16 @@ def gf(self, a):
 
     def E(self, a, order=0):
         if order == 0:
-            return self.efunc(1/a - 1.0)
+            return self.efunc(1./a - 1.0)
         else:
-            return self.efunc_prime(1/a - 1.0) * a
+            return self.efunc_prime(1./a - 1.0) * a
 
     def Hfac(self, a):
         return -2. - self.E(a, order=1) / self.E(a)
 
     def Om(self, a):
-        #assert abs(self.Om0 * a ** -3 / self.E(a) **2 - self.cosmo.Om(1.0 / a - 1)) < 1e-6
-        return self.cosmo.Om(1.0 / a - 1)
-        #return self.Om0 * a ** -3 / self.E(a) **2
+        z = 1./a-1
+        return self.cosmo.Omega_b(z) + self.cosmo.Omega_cdm(z) # non-relativistic 
 
```

Comment:
Is using only Omega_b + Omega_cdm right here, @rainwoodman ? It yields better agreement in the tests, due to the different Om() definitions, I think

Reply:
I think this is a good idea. The purpose of the ode solver is to describe the linear behavor of a traditional nbody simulation that that only includes perturbations of cdm and baryons.